### ISSUE TITLE
Help about section 4600

### ISSUE THREAD

**Issue by desk7** [ghacksuserjs/ghacks-user.js#902]
I'd like to have some information about section 4600.

I will create a new profile so I have nothing to reset. To achieve the best protection in Firefox for Windows, should I enable RFP and comment section 4600? Does the RFP replace all that prefs?

And when I will update user.js, will I have to check something in about:config? I didn't understand the sentence "RFP users should reset items in 4600 in about:config, as some cause RFP to not behave as you would expect and alter your fingerprint".  This means that every time I update user.js, I will have to manually check every entry? Are some enabled by default?

Thanks

**Comment by Thorin-Oakenpants** [comment]
> Does the RFP replace all that prefs

Yes, RFP replaces all of 4600 (and does more stuff as well). You do not need anything in section 4600, and some of them, if applied, change your fingerprint

---

- RFP is **enabled by default**
- Section **4600 is disabled by default** - it is wrapped in comment syntax

^^ you don't need to do anything if you want to be an RFP user

See in the pic how just above section 4600, the `blankWindow` pref is "active", and how the first pref in section 4600 is "inactive" - that's the "maxHardwareConcurrency" one - even though both do not have ` // ` in front of them.

That's because the whole section is wrapped in `/* */` comment syntax - note the red line down the left hand side of my editor, which shows I can fold/collapse it all.

See where the red line starts: that line starts with `/* ` .. if you change it to "//" then all of section 4600 becomes active: which you would do if you wanted to not use RFP but still have some fingerprinting protections.

![4600](https://user-images.githubusercontent.com/16656956/76332251-09870600-62e8-11ea-82b5-2c89b2afa053.png)

---

The comment about RFP users: i.e people who have `privacy.resistFingerprinting` = `true` is that **IF** they ever changed the prefs in section 4600 (manually, by flipping section 4600 on at some stage, whatever), then they should make sure all the prefs in 4600 are reset in about:config. Prefs don't reset themselves

If you're still confused, sing out

**Comment by desk7** [comment]
So, if I create a new empty profile and use user.js as is, I should have no problem, right? Even if Firefox updates, all the prefs in 4600 will have the default value, ie they stay reset. Will the default values be always false for RFP users?

**Comment by Thorin-Oakenpants** [comment]
> So, if I create a new empty profile and use user.js as is, I should have no problem, right?

Correct. Section 4600 is not applied, it is commented out. So in a new profile, those prefs in section 4600 are are default.

> Even if Firefox updates, all the prefs in 4600 will have the default value, ie they stay reset. Will the default values be always false for RFP users?

Firefox updates can change `default` values, but they do change prefs that are modified/user-set (you can tell they're modified because they are bold: ignoring internal ones set by Firefox itself such as update stamps etc). So any prefs in 4600 that Firefox changes the default value of, you would change too (because you should be at default as well). But, so would everyone else: so everyone is still the same in that regard. And if a pref in 4600 was to change it's default value and affect RFP, then RFP would probably consider that a bug

I seriously doubt any of the prefs in 4600 will change anyway

tl;dr: you'll be fine
